### PR TITLE
[AMSDK-8703] Update signal extension documentation

### PR DESCRIPTION
**Pull request by samthiru**
[AMSDK-8703] Update signal extension documentation

### CODE REVIEW

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+#### Java
+
+Add the  [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using the app's Gradle file:
+
+```
+implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+```
+
+Import the library:
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/README.md]: Import the Signal extension in your application class:

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### iOS
+
+1. Add the  [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods:
+
+![](./img/signal_config_objc.png)
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/README.md]: Let's use code samples instead of an image so it can be copy-pasted, see Analytics example https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics#add-analytics-to-your-app

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+#### Java
+
+```java
+public class MobiletApp extends Application {
+@Override
+public void onCreate() {
+super.onCreate();
+     MobileCore.setApplication(this);
+     try {
+         ...........................
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/README.md]: Let's also include configureWithAppId and MobileCore.start. Analytics is a good example here https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics#register-analytics-with-mobile-core

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+#### iOS
+
+Register the Signal extension in your app's `didFinishLaunchingWithOptions` function:
+
+**Objective-C**
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [ACPSignal registerExtension];
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/README.md]: Same comment for obj c and swift

File: using-mobile-extensions/mobile-core/signal/signals-api-reference.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Signal API reference
+
+
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/signals-api-reference.md]: Maybe we can move the extensionVersion API here and also mention that this extension listens for the information provided through collectPII API and add a link here.

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+#### Event details
+
+| **Event Type**                    | **Event Source**                      | **Paired** | **Paired Event** |
+| :-------------------------------- | :------------------------------------ | :--------- | :--------------- |
+| com.adobe.eventType.configuration | com.adobe.eventSource.responseContent | No         | N/A              |
+
+#### Data payload definition
+
+Definition of the key/value pairs that this listener should look for global.privacy in this event data.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: - Please reword to:
"The Signal extension will read the following keys from the configuration event:"
- Please add "global.privacy" as a key in the table (as per my convo with Emilia).

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+|                  | global.ssl                            | boolean    | If set to true, network requests should be dispatched via HTTPS. If false, network requests should be dispatched via HTTP |
+| Identity         | experienceCloud.org                   | string     | Experience Cloud Org Identifier                              |
+|                  | experienceCloud.server                | string     | Custom endpoint to be used for Visitor ID Service network requests |
+|                  | identity.adidEnabled                  | boolean    | If set to true, indicates that the SDK should expect a call to setAdvertisingIdentifier (removed v1.0.1) |
+| Lifecycle        | lifecycle.sessionTimeout              | number     | Number of seconds since the user background the app that must pass in order for the subsequent app launch (or resume) to be considered a new lifecycle session |
+| Rules Engine     | rules.url                             | string[]   | List of Rules endpoints (represented as strings), in order of preference |
+| Target           | target.clientCode                     | string     | Special identifier given to the customer by the Target product. |
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: I think we can just keep the keys that signal extension reads in this table, see example in Analytics https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-event-reference#data-payload-definition-2

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+|                  | target.environmentId                  | string     | Indicates the Environment ID for the Target customer         |
+|                  | target.timeout                        | string     | Amount of time, in seconds, to wait for a response from Target before timing out |
+
+#### Event data example
+
+Set Advertising Identifier request
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: This is a configuration change event.

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+```javascript
+{
+    "global.privacy": "true"
+}
+```
+
+## Events dispatched
+
+The Signal Extension does not dispatch any event.
+
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: Let's also mention that Signal extension doesn't share any shared state, section format example https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-event-reference#shared-states

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md (hunk 6 above)
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: Let's reword this to something like `The Signal extension reads the following keys from the configuration event:`

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+| **Event Type**                    | **Event Source**                      | **Paired** | **Paired Event** |
+| :-------------------------------- | :------------------------------------ | :--------- | :--------------- |
+| com.adobe.eventType.configuration | com.adobe.eventSource.responseContent | No         | N/A              |
+
+#### Data payload definition
+
+Definition of the key/value pairs that this listener should look for global.privacy in this event data.
+
+| Key       | Friendly name | Type                                                         | Optional | Description                                                  |
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: I think this table is not required, as all the events have this format. Let's keep only the data payload definition one.

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 11]
```diff
@@ -0,0 +1,95 @@
+# Signal event reference
+
+## Events handled
+
+### Rules Engine Response Content
+
+This event is used by the Signal extension to queue up and send network calls that correspond to the sync pii call or postback message.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: As per my Slack convo with @samthiru, please change to "`pii`".

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+
+```javascript
+{
+    "global.privacy": "true"
+}
+```
+
+## Events dispatched and Shared State
+
+The Signal Extension does not dispatch an event and also it doesn't share any shared state.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: - Please lower case "Extension" --> "extension" and reword the sentence as follows:
"The Signal extension does not dispatch an event and does not share any shared state."

File: resources/user-guides/signal-extension-and-rules-engine-integration.md [hunk 13]
```diff
@@ -63,15 +59,15 @@ ACPCore.collectPii(piiContextData)
...
 
-1. In Adobe Experience Platform Launch, create a Data element for the `user_email` context data key.
+1. In Adobe Experience Platform Launch, create a data element for the `user_email` context data key.
 
-![Data Element Example for Collect PII context data key](../../../.gitbook/assets/data_element_example_collect_pii.png)
+![Data Element Example for Collect PII context data key](img/data_element_example_collect_pii.png)
 
 1. In Experience Platform Launch, create a new rule for sending a postback.
 
 Create a new rule by selecting the Event Mobile Core Collect PII and the Action Mobile Core Send Postback as in the following image:
```
Comment (rekrishn) [inline resources/user-guides/signal-extension-and-rules-engine-integration.md]: Please reword as follows:
"Create a new rule by selecting the **Mobile Core Collect PII** event and the **Action Mobile Core Send Postback** action as illustrated in the following graphic:

File: resources/user-guides/signal-extension-and-rules-engine-integration.md [hunk 14]
```diff
@@ -63,15 +59,15 @@ ACPCore.collectPii(piiContextData)
...
 
-![Rule example using Collect PII event and Postback action](../../../.gitbook/assets/postback_pii_token_example.png)
+![Rule example using Collect PII event and Postback action](img/postback_pii_token_example.png)
 
 1. Use the above data element in the Postback action.
```
Comment (rekrishn) [inline resources/user-guides/signal-extension-and-rules-engine-integration.md]: Please reword to:
"Select the data element that you created in step 1 for the Postback action."

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 15]
```diff
@@ -0,0 +1,174 @@
+# Signal
+
+The Signal extension is bundled with the [MobileCore (Android)/ACPCore (iOS)](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/) extension and allows you to send postbacks to third-party endpoints and open URLs, such as web URLs or application deep links, when using rules actions in Adobe Experience Platform Launch. 
+
+To send PII data to external destinations, the `PII` action can trigger the Rules engine when certain triggers and traits match. When setting a rule, you can also set the `PII` action for a Signal event. The `collectPii` API can then be used to trigger the rule and send the PII data to a remote server.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: - Please change "Rules engine" to "Rules Engine" and ensure that the correct term is used throughout.

- Please change "When setting a rule" to "When creating a rule..."

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 16]
```diff
@@ -0,0 +1,174 @@
...
+{% endtabs %}
+
+### Register Signal extension
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+To call the set up methods that call the [setApplication\(\)](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#setapplication) method in the `onCreate()` method:
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: there are too many "methods" in this sentence! Please change the end of the sentence as follows:
"...method in `onCreate()`."

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 17]
```diff
@@ -0,0 +1,174 @@
...
+{% endtab %}
+{% endtabs %}
+
+##Define rules in Experience Platform Launch
+
+To define the rules in Experience Platform Launch, complete the steps in [Signal extension and Rules Engine integaration](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/signals/signals-extension-and-rules-engine-integration).
+
+##Implement the Mobile SDK to send PII data to external destinations
+
+To send PII data to external destinations, the `PII` action can trigger the Rules Engine when the configured triggers and traits match. When setting a rule, you can set the `PII` action for a Signal event, so that `collectPii` can trigger the rule and send the `PII` data..
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: - please remove the extra period at the end of the final sentence.
- Please replace "When setting a rule..." to "When creating a rule..."

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 18]
```diff
@@ -0,0 +1,174 @@
...
+
+For the information on collectPii and its usage, see collectPii in [AEP Mobile Core API reference Page](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii)
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: Please reword to:
"For more information about `collectPII` and its usage, see [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii).

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 19]
```diff
@@ -0,0 +1,174 @@
...
+
+For more information on how to configure the Signal postbacks in Adobe Experience Platform Launch refer [signals-extension-and-rules-engine-integration](https://aep-sdks.gitbook.io/docs/resources/user-guides/signals-extension-and-rules-engine-integration).
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: Please reword to:
"For more information about how to configure the Signal postbacks in Adobe Experience Platform Launch, see [doc link]().

File: using-mobile-extensions/mobile-core/signal/signal-api-reference.md [hunk 20]
```diff
@@ -0,0 +1,52 @@
...
+```jsx
+ACPSignal.extensionVersion().then(signalExtensionVersion => console.log("AdobeExperienceSDK: ACPSignal version: " + signalExtensionVersion));
+```
+
+{% endtab %}
+{% endtabs %}
+
+## CollectPII API
+
+The Signal extension listens for the information provided through [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii) API.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-api-reference.md]: Please add "the" between "through" and "[collectPII]()".

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 21]
```diff
@@ -0,0 +1,95 @@
+# Signal event reference
+
+## Events handled
+
+### Rules Engine Response Content
+
+This event is used by the Signal extension to queue up and send network calls that correspond to the [sync pii consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#sync-pii-consequence) or postback message.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: Please remove "up".

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 22]
```diff
@@ -0,0 +1,95 @@
...
+        ``"url"` `: ``"myApp://HomePage"
+    ``}
+}
+```
+
+
+
+### Configuration Response Content
+
+This event is dispatched on the Event Hub when a configuration change is processed.  If a change occurs, the listener updates the privacy status.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: looks like there might be an extra space between the sentences; there should only be 1 space.

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 23]
```diff
@@ -0,0 +1,95 @@
...
+
+#### Event details
+
+| **Event Type**                    | **Event Source**                      | **Paired** | **Paired Event** |
+| :-------------------------------- | :------------------------------------ | :--------- | :--------------- |
+| com.adobe.eventType.configuration | com.adobe.eventSource.responseContent | No         | N/A              |
+
+#### Data payload definition
+
+The Signal extension will read the following keys from the configuration event:
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: Please replace "will read" to "reads"

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 24]
```diff
@@ -0,0 +1,176 @@
...
+
+initSDK() {
+    ACPSignal.registerExtension();
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+##Define rules in Experience Platform Launch
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/README.md]: Let's remove this section as now it is included in the tutorials section.

File: using-mobile-extensions/mobile-core/signal/signal-api-reference.md [hunk 25]
```diff
@@ -0,0 +1,52 @@
...
+```jsx
+ACPSignal.extensionVersion().then(signalExtensionVersion => console.log("AdobeExperienceSDK: ACPSignal version: " + signalExtensionVersion));
+```
+
+{% endtab %}
+{% endtabs %}
+
+## CollectPII API
+
+The Signal extension listens for the information provided through the [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii) API.
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/signal/signal-api-reference.md]: The Signal extension can be used for handling collectPII rules, for more information check the Mobile Core collectPii<link> API.

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 26]
```diff
@@ -54,10 +54,9 @@ Import the Signal libraries:
 ```
 
 #### Swift
-
+In swift, the ACPCore includes ACPSignal
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: Please capitalize "swift" --> "Swift" and add a colon (:) after `ACPSignal`

File: using-mobile-extensions/mobile-core/signal/signal-api-reference.md [hunk 27]
```diff
@@ -48,5 +48,5 @@ ACPSignal.extensionVersion().then(signalExtensionVersion => console.log("AdobeEx
 
 ## CollectPII API
 
-The Signal extension listens for the information provided through the [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii) API.
+The Signal extension can be used for handling collectPII rules, for more information check the Mobile Core  [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii) API.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-api-reference.md]: Please reword as follows:
"The Signal extension can be used to handle `collectPII` rules. For more information, see the [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii) API."

File: using-mobile-extensions/mobile-core/signal/signal-event-reference.md [hunk 28]
```diff
@@ -18,15 +18,15 @@ This event is triggered by the Rules Engine when the events and conditions are m
...
-| triggeredconsequence | Description URL | Map<String, Object> | Yes      | Triggered Consequence details in the map. |
+| Key                  | Friendly name   | Type                | Optional | Description                                                  |
+| -------------------- | --------------- | ------------------- | -------- | ------------------------------------------------------------ |
+| triggeredconsequence | Description URL | Map<String, Object> | Yes      | Triggered Consequence details in the map.   <br />For the detailed description of each rule consequence refer the following pages. <br />[Postback consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#postback-consequence)<br/>[Sync PII consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#sync-pii-consequence)<br/>[Open URL consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#open-url-consequence) |
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/signal-event-reference.md]: In the 1st sentence (For the detailed...), please replace the period (.) with a colon (:).

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 29]
```diff
@@ -10,7 +10,7 @@ To get started with Signal extension, complete the following steps:
 
 1. Add the **Signal** extension to your app.
 2. Define the necessary rules in Experience Platform Launch. 
-3. (Optional) When using Send PII actions in Experience Platform Launch, implement the APIs to Experience Platform Launchcollect PII data and send it to the configured third-party destination.
+3. (Optional) When using Send PII actions in Adobe Launch, implement the SDK APIs to collect PII data and send it to the configured third party destination.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: Please change "Adobe Launch" --> Experience Platform Launch

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 30]
```diff
@@ -54,7 +54,7 @@ Import the Signal libraries:
 ```
 
 #### Swift
-In swift, the ACPCore includes ACPSignal
+In swift, the ACPCore includes ACPSignal:
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: please make "swift" upper case --> "Swift"

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 31]
```diff
@@ -0,0 +1,211 @@
...
+
+- When the `global.privacy` configuration is set to `optedout`, the Signals extension clears all queued hits and drops future network send requests.
+- If the configuration is `optunknown`, the network sends requests that are queued but not sent out.
+- If the configuration is `optedin`, the network requests to a third-party destination are allowed, provided the trigger conditions match.
+
+To get started with Identity, complete the following steps:
+
+1. Add the **Signal** framework to your app.
+2. Define the necessary rules in Adobe Experience Platform Launch. 
+3. Implement the SDK APIs to send PII data to external destinations.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: Don't think you need to say "SDK APIs" --> please change to either one or the other.

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 32]
```diff
@@ -0,0 +1,211 @@
...
+```jsx
+import {ACPSignal} from '@adobe/react-native-acpcore';
+```
+
+{% endtab %}
+{% endtabs %}
+
+## Register the Signal extension
+
+The `registerExtension()` API registers the Signal extension with the MobileCore extension. This API allows the extension to send and receive events to and from the Mobile SDK.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: In the 1st sentence, please add a space between "MobileCore" --> "Mobile Core"

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 33]
```diff
@@ -0,0 +1,211 @@
...
+    ACPSignal.registerExtension();
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+## Version of the Signal extension
+
+The `extensionVersion()` API returns the version of the Signal extension that is registered with the MobileCore extension.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: Please add a space in "MobileCore" --> "Mobile Core"

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 34]
```diff
@@ -0,0 +1,211 @@
...
+```jsx
+ACPSignal.extensionVersion().then(signalExtensionVersion => console.log("AdobeExperienceSDK: ACPSignal version: " + signalExtensionVersion));
+```
+
+{% endtab %}
+{% endtabs %}
+
+##Define rules in Adobe Experience Platform Launch
+
+Define the rules in AEP Launch by following the steps in the [Signal extension and Rules engine Integaration document](signals-extension-and-rules-engine-integration)
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: - Please replace "AEP Launch" with "Experience Platform Launch"
- Please change "engine" --> "Engine"
- Please lower case "Integration" --> "integration
- please remove "document" from the link text so that it reads "Signal extension and Rules Engine integration"

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 35]
```diff
@@ -0,0 +1,211 @@
...
+
+##Implement the SDK APIs to send PII data to external destinations
+
+To send PII data to external destinations, the `PII` action can trigger the Rules engine when certain triggers and traits match. When setting a rule, you can also set the `PII` action for a Signal event. The `collectPii` API can then be used to trigger the rule and send the PII data.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: please make the "e" in "engine" upper case --> "Engine"

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 36]
```diff
@@ -0,0 +1,211 @@
...
+
+For the information on collectPii and its usage, see collectPii in [AEP Mobile Core API reference Page](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii)
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: please reword to:
"For more information about `collectPII` and its usage, see [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii).

File: using-mobile-extensions/mobile-core/signal/README.md [hunk 37]
```diff
@@ -170,6 +170,6 @@ initSDK() {
 
 To send PII data to external destinations, the `PII` action can trigger the Rules Engine when the configured triggers and traits match. When creating a rule, you can set the `PII` action for a Signal event, so that `collectPii` can trigger the rule and send the `PII` data.
 
-For the information on collectPii and its usage, see collectPii in [AEP Mobile Core API reference Page](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii)
+For the information on `collectPii` and its usage, see [collectPii](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii)
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/signal/README.md]: Please change the beginning of the sentence to:
"For more information about `collectPII` and its usage..." and please add a period at the end of the sentence.
